### ISSUE TITLE
How do you create ignore sub in Contextual Chaining Substitutions?

### ISSUE THREAD

**Issue by waykyi** [fontforge/fontforge#4882]
I am trying to understand when to write ignore sub and the problem I have is that I don't know where exactly I can create them in FF. Can anyone explain why we need it and how and where I can create one in FF. I sort of understand there are exception cases in the Calt feature, where I need to write for it, right? Thanks

**Comment by KamilLanda** [comment]
I don't know I understand correctly to your ask, but if I need some exception, I use **empty** Single Lookup for sustitution. Fox example I made Contextual Chainign Substituion for big letters 'ABCD' followed by small letters 'abcd'. If first letter is big and second letter is small, then it makes substitution for second letter and changes it to big one. Exception is for combination 'Cc' -> don't change 2nd 'c' to 'C'. And for this exception I used the **empty** Single Lookup *(in .sfd file named 'single' ignore)*.  
Example: [ignore.sfd.zip](https://github.com/fontforge/fontforge/files/7838448/ignore.sfd.zip)

**Comment by waykyi** [comment]
Thank you very much for your .sfd file and I was able to see your sample. I learned a new way in which your use the empty lookup. Thanks for taking your time.

**Comment by ctrlcctrlv** [comment]
If you are at this level of OTL complexity you should absolutely be using FEA files and not the GUI.

**Comment by waykyi** [comment]
> If you are at this level of OTL complexity you should absolutely be using FEA files and not the GUI.

Yes ctrlcctrlv, I am still learning FF and though I know some FEA writing, I prefer to use GUI in FF. Now I think, I was able to use GUI to create ignore sub now. I use coverage table and add glyphs orderly without assigning any lookup. Which FF write out ignore sub and behave as exception. FEA writing in Chaining Substitutions with classes, import them into FF turn out as coverage table :( . But still work what I intended. In fact I like to use GUI to get lot of FEAs for my font without leaving FF. I like easy use of GUI without knowing FEA writings. Thanks
